src/Slide/Slide.jsx

```diff
@@ -116,6 +119,18 @@ const Slide = class Slide extends React.PureComponent {
       tempStyle.paddingBottom = pct((naturalSlideHeight * 100) / naturalSlideWidth);
     }
 
+    const innerStyle = {};
+    if (intrinsicAxisSize) {
+      if (orientation === 'horizontal') {
+        tempStyle.height = 'unset';
+      } else {
+        tempStyle.width = 'unset';
+      }
+      tempStyle.position = 'unset';
+      tempStyle.paddingBottom = 'unset';
+      innerStyle.position = 'unset';
+    }
```

Comment:
Question on this: instead of setting CSS via JavaScript, are we able to override the existing CSS with CSS?